tools/python/boutiques/test.py

```diff
@@ -15,6 +15,7 @@ def test(descriptor, test, invocation):
     ret = bosh.execute("launch",
                        descriptor,
                        invocation.name)
+    print(ret)
```

Comment:
Do we want this print statement, or was it for debugging?

Reply:
Both :smiley: It is useful to have a more verbose stdout in tests when they fail. In case it's unclear, here we're talking about tool tests launched by `bosh test`.